lstchain/scripts/onsite/onsite_create_calibration_file.py

```diff
@@ -78,6 +79,7 @@ def none_or_str(value):
 config_file = args.config
 mongodb = args.mongodb
 yes = args.yes
+pro_symlink = not args.no_pro_symlink
```

Comment:
> For me, it is not easy to see how this will work with different versions.
> 
> Imagine we are in `v1`, we produce all the calibrations without using `--no_pro_symlink`, this will create symlinks to `pro`:
> 
> ```
> drs4_baseline
>   v1
>   pro -> v1
> calibration
>   v1
>   pro -> v1
> drs4_time_sampling_from_FF
>   v1
>   pro -> v1
> ffactor_systematics
>   v1
>   pro -> v1
> ```
> 
> Imagine now, you are in `v2` and wants to produce a new `drs4_baseline` and `calibration` using the current `pro -> v1` in `drs4_time_sampling_from_FF` and `ffactor_systematics`. If you assume the default in `--no_pro_symlink`, drs4_baseline `pro` folder will be updated with the new version and point to `pro -> v2`. The directory assumed for `drs4_time_sampling_from_FF` and `ffactor_systematics` will be `pro -> v1` which is what we want I think. The directory tree would look like this afterward:
> 
> ```
> drs4_baseline
>   v1
>   v2
>   pro -> v2
> calibration
>   v1
>   v2
>   pro -> v2
> drs4_time_sampling_from_FF
>   v1
>   pro -> v1
> ffactor_systematics
>   v1
>   pro -> v1
> ```
> 
> If this is how it is suppose to work I think I'm fine with it

yes, this is the idea.